gensrc/thrift/FrontendService.thrift

```diff
@@ -523,6 +523,8 @@ struct TStreamLoadPutRequest {
     25: optional i64 thrift_rpc_timeout_ms
     26: optional string json_root
     27: optional bool partial_update
+    28: optional string transmissionCompressionType
```

Comment:
1. Already have a compression type `enum TCompressionType` define in gensrc/thrift/Types.thrift
2. naming should accord to other fields, better to use `transmission_compression_type`

Reply:
do we have any rule of naming, there have both underscore and camel naming in this structure

Reply:
Before if the struct is used by BE this use underscore like cpp, if the struct is used by FE this use camel like Java.

Reply:
The problem is that thrift is used by BE and FE both. I will use the underscore to name it uniformly.